src/hooks/search/utils.ts

```diff
@@ -218,6 +238,43 @@ const itemInFilterGroup = (tags: Tag[], filters: FilterItem[]) => {
     );
 };
 
+export const hasEmptyFilterAndQuery = (
+    searchString: string,
+    allFilters: FilterItem[]
+) => {
+    return (!searchString || searchString == '') && allFilters.length == 0;
+};
+
+// TODO: Refactor.
+export const getResultData = (
+    data: ContentItem[],
+    initialSearchData: ContentItem[] | undefined,
+    searchString: string,
+    allFilters: FilterItem[],
+    initialPageNumber: number,
+    initialPageResetFlag: boolean
+) => {
+    return initialSearchData && hasEmptyFilterAndQuery(searchString, allFilters)
+        ? initialSearchData
+        : data.slice(
+              !initialPageResetFlag && initialPageNumber > 1
+                  ? (initialPageNumber - 1) * DEFAULT_PAGE_SIZE
+                  : 0
+          );
+};
+
+export const getResultIsValidating = (
+    initialSearchData: ContentItem[] | undefined,
+    searchString: string,
+    allFilters: FilterItem[],
+    earchString: string,
```

Comment:
Looks like `earchString` was accidentally left in here

Reply:
Oh wow, haha. Somehow all of this was still working properly too.